app/src/ui/rebase/choose-branch.tsx

```diff
@@ -87,15 +87,15 @@ export class ChooseBranchDialog extends React.Component<
       initialBranch
     )
 
-    if (selectedBranch !== null) {
-      this.onBranchChanged(selectedBranch)
-    }
-
     this.state = {
       selectedBranch,
       rebasePreview: null,
       filterText: '',
     }
+
+    if (selectedBranch !== null) {
+      this.onBranchChanged(selectedBranch)
```

Comment:
So this *might* still end up calling `setState` before the component is mounted. Given that we're using a `promiseWithMinimumTimeout` it seems very unlikely but still, should we just move this call to `componentDidMount`?

Reply:
@niik works for me

Reply:
Addressed in afce9ea7e